resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala

```diff
@@ -36,6 +36,12 @@ private[spark] object Config extends Logging {
       .stringConf
       .createOptional
 
+  val KUBERNETES_INTERNAL_MASTER =
```

Comment:
Maybe, shall we add `_URL` at the end of the variable?

Reply:
done